### DISCUSSION TITLE
when  will  windows be supported

### DISCUSSION THREAD

**Discussion by Super1Windcloud** [zed-industries/zed#13911] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

when  will  windows be supported

**Comment by SomeoneToIgnore** [comment]
Consider asking generic questions in https://github.com/zed-industries/zed/discussions next time.

Windows support is released when the time comes.

Currently, no work in that area is done or planned by Zed team, the whole development happens thanks to our great community contributors.

**Comment by FinnT730** [comment]
If you know how to compile Rust, you can compile it yourself. I have seen some already do it, but they have not shared the executable.
100% possible to do from source, but some things might not work well

**Reply by jackTabsCode** [reply]
A guide is provided here:
https://zed.dev/docs/development/windows

It's pretty simple to build Rust applications! I recommend you build for release.

**Reply by ZeyadMoustafaKamal** [reply]
Yeah I did it and it works very well. Doesn't this mean that Windows is actually supported or what do you think ?

**Reply by SomeoneToIgnore** [reply]
I think you should define "supported" and do a small research before asking: 

* https://github.com/zed-industries/zed/pulls?q=is%3Apr+is%3Aopen+windows
* https://github.com/zed-industries/zed/issues?q=is%3Aissue+is%3Aopen+windows

Zed team does not support Windows and does not provide Windows builds or auto updates.

But it does work under Windows indeed, and indeed very well in comparison to the previous year.

**Reply by jackTabsCode** [reply]
> Yeah I did it and it works very well. Doesn't this mean that Windows is actually supported or what do you think ?

It's not supported. It does work, but not all of the features are there, and it crashes somewhat frequently.

**Comment by malisipi** [comment]
You can install Zed from msys2 as prebuild binary.
A guide is here: https://zed.dev/docs/development/windows#installing-from-msys2

Since Windows is not supported as officially, you may encounter with some bugs.

**Comment by herawais** [comment]
We dont care about windows, we need for linux

**Comment by c16a** [comment]
@herawais you can build Zed for Linux yourself. https://zed.dev/docs/development/linux

**Comment by deevus** [comment]
There are also nightly builds available via [Scoop](https://scoop.sh/):

```
scoop bucket add versions
scoop install versions/zed-nightly
```

**Reply by jmoraispk** [reply]
Best answer. Took me <5min. Thanks.

**Reply by jmoraispk** [reply]
If a Zed developer passes by... please add this to the instructions.
